CHANGELOG.md

```diff
@@ -7,9 +7,17 @@ please see [changelog_updates.md](docs/dev/changelog_updates.md).
 
 ### Overview
 
+Starting from version `8`, the Broker has been merged with the Community edition.
```

Comment:
Maybe this doesn't need to marked as a major change.

For the user, the only news is that the broker's versioning line changes.
The feature set remains the same.

Reply:
We definitely should

Because once the refactoring is done, we should do a major release where we align all the versions (connector, UI, broker) to the common next highest major version, of course with an explanation why versions were skipped.

this is for the strategic goal of aligning versions

a TODO for the next release would be then to also adjust the version VARs in .env files, release processes, (future)  changelog entries so that we can have a streamlined "one community edition version"

Reply:
Noted for next release
https://github.com/sovity/edc-extensions/issues/928#issuecomment-2117525942